src/applications/hca/enrollment-status-helpers.jsx

```diff
@@ -98,7 +98,7 @@ export function getEnrollmentDetails(
     blocks.push(
       <>
         <strong>We enrolled you on: </strong>
-        {moment(enrollmentDate).format('MMMM D, YYYY')}
+        {format(Date.parse(enrollmentDate), 'MMMM d, yyyy')}
```

Comment:
`enrollmentDate` has the same story as `applicationDate`.  It may be wise to add some unit tests for this function.